### DISCUSSION TITLE
Recaptchas

### DISCUSSION THREAD

**Discussion by tparency** [CMSgov/price-transparency-guide#144] [Q&A] [answered]
As we realize the file sizes of the mandated files and to control costs of hosting massively large downloadable files. Can plans use a recaptcha as a front door on a page to limit "bad actors" and errant scripting?

**Comment by jonberkery-work** [comment]
I wondered the same thing.  Can an automated "web crawler" handle a recaptcha question?

Note: I am no authority on the rule, I work for a payer doing the diligence to protect us from bringing down our entire file hosting site.

**Comment by shaselton-usds** [comment, marked as answer]
At this time recaptchas would not be allowed as they would interfere with automated searches.

**Reply by awm33** [reply]
At least one payer implemented one

https://individual.carefirst.com/individuals-families/mandates-policies/machine-readable-file.page

In general, if we notice guideline violations, should we report them somehow? I would try to contact the payers first, but I have yet to come across any sort of contact or feedback information, which I understand why they wouldn't want to.

**Reply by dgolden** [reply]
I recently ask CMS, and their response is below:

This is an update on your Health Plan Price Transparency inquiry. Please find our response below. 
PT Case Number: 7175814080361
Status: Answer communicated to submitter 

Health Plan Price Transparency case summary: Does CMS have a directory for contacting the payor or TPA, to advise them there is a problem with an MRF?

Response: Please file a complaint with additional detail, such as plan name and file internet location, via our webform here: https://www.cms.gov/healthplan-price-transparency/contact-us
Thank you.

**Reply by dgolden** [reply]
My 2 cents is that the next version of the schema specification should have a required field in the header section of schema that provides contact email for whoever created the MRF. In addition to reporting issues, there is a bonafide need for Q&A and suggestions on how to improve reporting. Having all information flow go through an anonymous chokepoint (CMS contact email) is a bad system design. I do realize that CMS has an important role and vested interest in checking for compliance ...
